api-spec/wallet-api.yaml

```diff
@@ -130,9 +130,6 @@ components:
           description: Credit card expiry date. The date format is `YYYYMM`
           pattern: ^\d{6}$
           example: '203012'
-        contractNumber:
```

Comment:
Why are you moving out contract number from card details and put it on the request root?